crud/select/merger.lua

```diff
@@ -58,17 +58,7 @@ end
 
 local data = ffi.new('const unsigned char *[1]')
 
-local function decode_response_headers(buf)
-    -- {48: [cursor, [tuple_1, tuple_2, ...]]} (exactly 1 pair of key-value)
-    data[0] = buf.rpos
-
-    -- 48 (key)
-    data[0] = data[0] + 1
-
-    -- [cursor, [tuple_1, tuple_2, ...]] (value)
-    data[0] = data[0] + 1
-
-    -- Decode array header
+local function decode_response_array_header()
```

Comment:
This change definitely needs some CHANGELOG entry

Reply:
Added changelogs for all changes.